django/db/models/expressions.py

```diff
@@ -483,6 +483,8 @@ def __init__(self, value, output_field=None):
         self.value = value
 
     def as_sql(self, compiler, connection):
+        if self.value is None:
```

Comment:
Could you add a comment to explain the Oracle corner case? It would be good to know it without using `git blame` :)